### DISCUSSION TITLE
How to load a MF6 model and change name/ws

### DISCUSSION THREAD

**Discussion by giovannifi** [modflowpy/flopy#1797] [Q&A]
I build and run a calibration MF6 model with flopy and now I would like to create a predictive simulation. My idea is to:
- load the calibration model
- rename sim, GWF, GWT
- change the workspace for the simulation and GWF and GWT

to do that I am doing the following but it is not working:

```
name_tr = 'model_calib'
name_pr = 'model_pred'
workspace_tr = r'c:\model_calibration'
workspace_pr = r'c:\model_prediction'
sim = flopy.mf6.MFSimulation.load(sim_name=name_tr, exe_name=path_exe, sim_ws=workspace_tr, version="mf6")

sim.name = name_pr

sim.set_sim_path = workspace_pr
gwf = sim.get_model(name_tr + '_gwf')
gwt = sim.get_model(name_tr + '_gwt')
gwf.set_model_relative_path(workspace_pr)
gwt.set_model_relative_path(workspace_pr)
gwf.name = name_pr + '_gwf'
gwt.name = name_pr + '_gwt'
gwf.name_file.name = name_pr + '_gwf.nam'
gwt.name_file.name = name_pr + '_gwt.nam'

sim.write_simulation()
```

When I run the above, I have the files mfsim.nam, ats, tdis and ims saved in the old workspace. The gwt and gwf files saved in the new folder, but they are saved with the old root name. Even the "sim" files are saved with the old root name. To me it seems that the following commands are ineffective:

```
sim.name = name_pr
sim.set_sim_path = workspace_pr
gwf.name = name_pr + '_gwf'
gwt.name = name_pr + '_gwt'
gwf.name_file.name = name_pr + '_gwf.nam'
gwt.name_file.name = name_pr + '_gwt.nam'
```

Any help would be appreciated

**Comment by spaulins-usgs** [comment]
FloPy is not saving all files to your new workspace folder because "set_sim_path" is a method (not a property).  Try changing:

    sim.set_sim_path = workspace_pr

To:

    sim.set_sim_path(workspace_pr)

To rename your simulation's files, try setting the "filename" attribute of each package or nam file you want to rename.  For example,

    gwf.name_file.filename = name_pr + '_gwf.nam'

**Comment by jdhughes-dev** [comment]
An example of loading a model, changing the workspace, and writing the files to the new workspace is given in https://flopy.readthedocs.io/en/latest/Notebooks/mf6_parallel_model_splitting_example.html#Example-1:-splitting-a-simple-structured-grid-model
